app/controllers/pets_controller.rb

```diff
@@ -1,2 +1,33 @@
 class PetsController < ApplicationController
+  def index
+    @pet = Pet.all
+  end
+
+  def create
+    @pet = Pet.new(params[:pet])
+    @pet.save
+    # Will raise ActiveModel::ForbiddenAttributesError
+    redirect_to pet_path(@pet)
+  end
+
+  def edit
```

Comment:
do we need to edit ? did we  project this? (i am also happy to do it ... i am just enjoing reviewing the path)